scripts/bootstrap.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const os = require('os');
+const fs = require('fs');
+
+const symlinkType = os.platform() === 'win32' ? 'junction' : 'dir';
+
+const repoPath = path.resolve(__dirname, '..');
+const extensionspath = path.resolve(os.homedir(), '.vscode', 'extensions');
+const disabledPath = path.join(extensionspath, 'disabled');
+const bootstrapedDraculaPath = path.join(extensionspath, 'dracula');
+
+const commands = {
+    attach() {
+        const draculaDir = fs

```

Comment:
```suggestion
        if (fs.existsSync(disabledPath)) {
            return;
        }
        const draculaDir = fs
```

Reply:
The diff here is super strange... I've just applied!